maqs-selenium/src/test/java/com/cognizantsoftvision/maqs/selenium/SeleniumTestObjectUnitTest.java

```diff
@@ -13,7 +12,7 @@
 /**
  * The type Selenium test object test.
  */
-public class SeleniumTestObjectUnitTest extends BaseGenericTest {
+public class SeleniumTestObjectUnitTest extends BaseSeleniumTest {
```

Comment:
Why add a specific base when generic is all we need?

Reply:
reverted back to baseGeneric